tf03_driver/launch/four_sensors.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
+
+    <group ns="tf03_driver">
+        <rosparam param="can_transmit_id"> [3,4,5,6]</rosparam>
+        <rosparam param="sensor_frame"> ["front_left", "front_right", "rear_left", "rear_right"]</rosparam>
+        
+        <rosparam param="sensor_interface">can</rosparam>
+        <rosparam param="can_device">benewake_can</rosparam>
```

Comment:
Can devices not really supported at the moment so it is the best to get rid of it